src/plugins/crypto/README.md

```diff
@@ -109,62 +110,62 @@ Set the CMake variables `OPENSSL_INCLUDE_DIR` and `OPENSSL_LIBRARIES` to your de
 
 The crypto plugin will encrypt and decrypt values using AES-256 in CBC mode.
 
-The key derivation is still WIP.
+The GPG binary is required for handling the cryptographic keys.
 
-### Planned Features ###
+At the moment the plugins will only run on UNIX/Linux-like systems, that implement `fork ()`.
 
-- Encryption of values
-- Decryption of values
-- Key derivation by metadata (password provided within a meta-key)
-- Key derivation by using a specified key-file (like the SSH client does)
-- Key derivation by utilizing the pgp-agent
+## Examples ##
+
+To mount a backend with the gcrypt plugin variant that uses the GPG key 9CCC3B514E196C6308CCD230666260C14A525406, use:
 
-The encryption and decryption of values is a straight forward process, once the key and IV are supplied.
-The key-derivation process in a library is a bit tricky.
+	kdb mount test.ecf user/t crypto_gcrypt "crypto/key=9CCC3B514E196C6308CCD230666260C14A525406"
 
-The crypto plugin itself can hold configuration data about the order of the applied key derivation functions.
-For example, if a password is provided by a meta-key, it will be used, otherwise we look for a key file.
-If no key file has been configured, we try to trigger the PGP agent, etc.
+Now you can specify a key `user/t/a` and protect its content by using:
 
-The following example configuration illustrates this concept:
+	kdb set user/t/a
+	kdb setmeta user/t/a crypto/encrypt 1
+	kdb set user/t/a "secret"
```

Comment:
There seems to be a problem if you use the spec-keys instead:
```
$ kdb mount test.ecf /t crypto_gcrypt crypto/key=0b2f0ddc
$ kdb global-mount spec
$ kdb setmeta spec/t/a crypto/encrypt 1
$ kdb set /t/a "secret"                
The command set failed while accessing the key database with the info:
Error (#130) occurred!
Description: The configuration is invalid or incomplete.
Ingroup: plugin
Module: crypto
At: /home/markus/Projekte/Elektra/current/src/plugins/crypto/gcrypt_operations.c:107
Reason: missing salt as meta-key crypto/salt for key spec/t/a
Mountpoint: user/t
Configfile: /home/markus/.config/test.ecf
```

(The error is also vague)